lms/static/scripts/frontend_apps/components/Breadcrumbs.js

```diff
@@ -0,0 +1,48 @@
+import { LinkButton } from '@hypothesis/frontend-shared';
+import { createElement } from 'preact';
+
+/**
+ * @template Item
+ * @typedef BreadcrumbProps
+ * @prop {Item[]} items
+ * @prop {(i: Item) => void} onSelectItem
+ * @prop {(i: Item) => any} [renderItem]
```

Comment:
It maybe useful to add a comment about what is the purpose of this prop.
```suggestion
 * @prop {(i: Item) => any} [renderItem] - function to transform the Item into a renderable component
```